.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: "CI"
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+
+# For auto-approving/merging PRs.
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  check:
+    name: "Checks and Tests"
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [14.x, 16.x, 18.x]
+
+    steps:
+      - name: "Run Yarn Basic Checks"
+        uses: SkynetLabs/.github/.github/actions/yarn-basic-checks@master
```

Comment:
`/.github/.github/` seems weird but probably ok?

Reply:
Yeah, that's where the actions are located but I don't know why we organized it like that exactly.